### PR TITLE
[TESTS] Added simple upgrade test

### PR DESCRIPTION
**Pull request by antdanchenko**
* added simple upgrade test
* changed apk name of nightly builds on sauce to reflect common name pattern (StatusIm-190115-125934-dd31ee-e2e.apk instead of im.status.ethereum-e2e-dd31ee.apk)

### CODE REVIEW

**Comment by antdanchenko** [comment]
@jakubgs please review changes in  [ci/android.groovy](https://github.com/status-im/status-react/pull/7279/files#diff-921ddb8a284f72367049b92eff26b9b6)

File: ci/android.groovy [hunk 1]
```diff
@@ -28,7 +28,7 @@ def uploadToSauceLabs() {
   if (changeId != null) {
     env.SAUCE_LABS_NAME = "${changeId}.apk"
   } else {
-    env.SAUCE_LABS_NAME = "im.status.ethereum-e2e-${GIT_COMMIT.take(6)}.apk"
+    env.SAUCE_LABS_NAME = "${pkg}"
```
Comment (jakubgs) [inline ci/android.groovy:44]: Where does `pkg` come from in that context? I don't see it's source.

**Review by jakubgs — CHANGES_REQUESTED** [review]

File: ci/android.groovy (hunk 1 above)
Comment (antdanchenko) [reply ci/android.groovy:44]: from https://github.com/status-im/status-react/pull/7279/files/8eedc83cc57ef5ffd826dc3abab5c5a6b07655e4#diff-921ddb8a284f72367049b92eff26b9b6R12
Or I can't use this in uploadToSauceLabs?

Comment (jakubgs) [reply ci/android.groovy:44]: Nope, it's a context of a different method, so I doubt that will work.

Comment (antdanchenko) [reply ci/android.groovy:44]: It seems that I should call `cmn.pkgFilename(type, 'apk')`, right?

Comment (jakubgs) [reply ci/android.groovy:44]: Yeah, that should work. I don't think passing that from the original place would make for nice code, that should be just fine.

Comment (antdanchenko) [reply ci/android.groovy:44]: i've added the definition of pkg to the method

**Review by jakubgs — APPROVED** [review]
lgtm

**Review by annadanchenko — APPROVED** [review]

**Review by Serhy — APPROVED** [review]

**Pull request merged**